docs/badges/umd.svg

```diff
@@ -14,7 +14,7 @@
   <g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="11">
     <text x="16.5" y="15" fill="#010101" fill-opacity=".3">umd</text>
     <text x="16.5" y="14">umd</text>
-    <text x="59" y="15" fill="#010101" fill-opacity=".3">46.63 kB</text>
-    <text x="59" y="14">46.63 kB</text>
+    <text x="59" y="15" fill="#010101" fill-opacity=".3">25.79 kB</text>
+    <text x="59" y="14">25.79 kB</text>
```

Comment:
this is an artifact of one of dependencies being moved to dev-dependencies (where it should have been).